src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java

```diff
@@ -204,23 +205,19 @@ void setUp()  {
             + "should return the case name with a with 200 response code")
     @Test
     void testPostRequestServiceHearingValues() throws Exception {
-        ServiceHearingValues model = ServiceHearingValues.builder()
-                .caseNamePublic(CASE_NAME)
-                .autoListFlag(false)
-                .hearingIsLinkedFlag(true)
-                .build();
-        String json = asJsonString(model);
-
         ServiceHearingRequest request = ServiceHearingRequest.builder()
                 .caseId(String.valueOf(CASE_ID))
                 .build();
 
+        String actualJson = ResourceLoader.loadJson("serviceHearingValuesForControllerTest.json");
+        ServiceHearingValues serializedObject = mapper.readValue(actualJson, ServiceHearingValues.class);
```

Comment:
What is this object being used for?

Reply:
not used will be removed. thanks for spotting this